families/families.js

```diff
@@ -7,39 +7,54 @@ import {
 
 checkAuth();
 
-const familiesEl = document.querySelector('.families-container');
+const familyHold = document.querySelector('.families-container');
 const logoutButton = document.getElementById('logout');
 
 logoutButton.addEventListener('click', () => {
     logout();
 });
 
-function displayFamilies() {
+async function displayFamilies() {
```

Comment:
You have made `displayFamilies()` an async function, but I don't see where you are awaiting anything.